### PR TITLE
[fix]add config parse for degree algorithm

### PR DESCRIPTION
**Pull request by fbpcchen**
…ic 算法上跑通

## What type of PR is this?
- [ ] bug
- [ ] feature
- [ ] enhancement

### CODE REVIEW

**Comment by wey-gu** [comment]
Thanks @fbpcchen for the contribution :)

**Comment by fbpcchen** [comment]
> Thanks @fbpcchen for the contribution :) could you please sign the CLA?

signed.

**Review by wey-gu — APPROVED** [review]

File: nebula-algorithm/src/main/scala/com/vesoft/nebula/algorithm/utils/DecodeUtil.scala [hunk 1]
```diff
@@ -53,6 +53,17 @@ object DecodeUtil {
     (encodeDataframe, encodeId)
   }
 
+  def convertEncodeId2StringId(dataframe: DataFrame, encodeId: DataFrame): DataFrame = {
```
Comment (Nicole00) [inline nebula-algorithm/src/main/scala/com/vesoft/nebula/algorithm/utils/DecodeUtil.scala:56]: What's the difference between convertEncodeId3StringId and converAlgoId2StringId

Comment (fbpcchen) [reply nebula-algorithm/src/main/scala/com/vesoft/nebula/algorithm/utils/DecodeUtil.scala:56]: I re-examined and tested the code, and found that it is indeed the same. 
It should be a misjudgment caused by the problem of receiving the encodeId parameter of the degree_static algorithm. 
Let me resubmit the code that solves the algorithm encodeId configuration parameter passing problem.

**Review by Nicole00 — DISMISSED** [review]
LGTM

**Comment by Nicole00** [comment]
Hi @fbpcchen , looks like the cla does not take effect, could you please re-sign the cla and re-push?

**Comment by wey-gu** [comment]
@fbpcchen maybe you should modify the commit's git author(both name and email) on the local branch and push force.

> 陈燃 seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

**Comment by fbpcchen** [comment]
> @fbpcchen maybe you should modify the commit's git author(both name and email) on the local branch and push force.
> 
> > 陈燃 seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

done

**Comment by fbpcchen** [comment]
> Hi @fbpcchen , looks like the cla does not take effect, could you please re-sign the cla and re-push?

It is the authentication problem of the commit I submitted , please review it again.

**Review by wey-gu — APPROVED** [review]

**Pull request merged**